build/plugin-queued-callbacks-and-voicemail.js

```diff
@@ -0,0 +1,3 @@
+/*! For license information please see plugin-queued-callbacks-and-voicemail.js.LICENSE.txt */
```

Comment:
remove this file, and also add `build/` to the `.gitignore`

Reply:
Solved in c867f67